### PR TITLE
Refactor(Client+Server): rename RedesignProject flow types and params

### PR DESCRIPTION
**Pull request by yuval-hazaz**
Close: #7929

## PR Details

Refactor model organizer flows with new names and types. No actual logic changes.

### Server side
- Renamed mutation copiedEntities to redesignProject, including RedesignProjectArgs and RedesignProjectInput
- Renamed "ModelGroupResource" type to "RedesignProjectNewService"
- Renamed "CpiedEntity"  type to "RedesignProjectMovedEntity"

### Client side
- Update names based on server changes
- Rename local params based on server changes
- Show detailed error message after ApplyChanges clicked

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by morhag90 — APPROVED** [review]

File: packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizerToolbar.tsx [hunk 1]
```diff
@@ -63,16 +64,18 @@ export default function ModelOrganizerToolbar({
...
     setConfirmChanges(!confirmChanges);
-    setCreateError(false);
+    setApplyChangesErrorMessage(null);
   }, [confirmChanges, setConfirmChanges]);
 
   useEffect(() => {
-    setCreateError(createEntitiesError);
-  }, [createEntitiesError, setCreateError]);
+    setApplyChangesErrorMessage(formatError(applyChangesError));
+  }, [applyChangesError]);
```
Comment (mulygottlieb) [inline packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizerToolbar.tsx:78]: Don't you need to add `setApplyChangesErrorMessage` to the useEffect deps?

File: packages/amplication-client/src/Project/ArchitectureConsole/hooks/useModelOrganizer.tsx [hunk 2]
```diff
@@ -476,9 +472,8 @@ const useModelOrganization = ({ projectId, onMessage }: Props) => {
       nodes.push(newResourceNode);
 
       const newService = {
-        tempId: newResource.tempId,
+        id: newResource.tempId,
```
Comment (mulygottlieb) [inline packages/amplication-client/src/Project/ArchitectureConsole/hooks/useModelOrganizer.tsx:475]: Wouldn't this be `newResource.id` ?

File: packages/amplication-server/src/core/resource/resource.service.ts [hunk 3]
```diff
@@ -565,7 +565,7 @@ export class ResourceService {
       };
 
       const resource = await this.createService(args, user);
-      resource.tempId = modelGroupResource.tempId; //@todo: remove tempId from resource type
+      resource.tempId = newService.id; //@todo: remove tempId from resource type
```
Comment (mulygottlieb) [inline packages/amplication-server/src/core/resource/resource.service.ts:568]: Perhaps this answers my previous comment. Why didn't you remove tempId completely?

File: packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizerToolbar.tsx (hunk 1 above)
Comment (mulygottlieb) [reply packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizerToolbar.tsx:78]: image

**Review by mulygottlieb — APPROVED** [review]

**Pull request merged**